weave/trace_server/interface/base_object_classes/actions.py

```diff
@@ -15,19 +15,19 @@ class LlmJudgeActionSpec(BaseModel):
     response_schema: dict[str, Any]
 
 
-class ContainsWordsActionSpec(BaseModel):
+class ContainsWordsActionConfig(BaseModel):
     action_type: Literal["contains_words"] = "contains_words"
     target_words: list[str]
 
 
-ActionSpecType = Union[
-    LlmJudgeActionSpec,
-    ContainsWordsActionSpec,
+ActionConfigType = Union[
+    LlmJudgeActionConfig,
+    ContainsWordsActionConfig,
 ]
 
 
-class ActionDefinition(base_object_def.BaseObject):
-    spec: ActionSpecType = Field(..., discriminator="action_type")
+class ActionSpec(base_object_def.BaseObject):
+    config: ActionConfigType = Field(..., discriminator="action_type")
```

Comment:
Changed the nested property to `config` so that we don't have a nested "spec". This also matches the parameters in the dispatcher